src/index.html

```diff
@@ -11,6 +11,43 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      class="searchbar large"
```

Comment:
```suggestion
      class="searchbar"
```

Reply:
Thank you, guys. But could you please show all the issue in my code initially just for saving our time and fast completing the task?

Reply:
if smth is unclear just ask in fe_chat0 and tag mentors - it will save your time

Reply:
no, I thought that everything was fine, but in fact there were some issues in my code after checking. I mean it woud be great if  all the issues and defects are shown straight-away after checking

Reply:
I guess it is clear examples in checklist
We won't write code instead of you - just highlight the mistakes
and if smth is unclear - you always may ask in chat.
if you mean, that this issue wasn't detected initially - sorry. We do all our best and try to provide good review, but we are people too and can miss smth initially, unfortunately. It was not intentionally

Reply:
Sure, I understand it. Thank you so much for your help)

Reply:
If I offended you, I do apologize for that. In no case did not want to do it